### ISSUE TITLE
General error during semantic analysis: Unsupported class file major version 64

### ISSUE THREAD

**Issue by regehr** [pcminer-tools/pcminer#39]
I'm trying to update the database to include PLDI '23 on a relatively up-to-date M1 Macbook, and `./gradlew run` gives me a lot of error output with this at the start:

```
> startup failed:
  General error during semantic analysis: Unsupported class file major version 64
```

my machine has Java 20 on it and I suspect I can fix this by downgrading back to 19, but is there a way to fix this for everyone? I don't really do Java at all, apologies if this is silly.

**Comment by msridhar** [comment]
Hi @regehr can you try with the `latest-gradle` branch I just pushed and let us know?  I think that might fix it.

**Comment by regehr** [comment]
works!! thanks Manu!

(also, downgrading openjdk 19 wasn't enough -- that just said "major version 63" instead of 64 :)

**Comment by msridhar** [comment]
Fixed by #40
